omnibus/config/software/datadog-agent-integrations-py2.rb

```diff
@@ -171,14 +176,26 @@
     #
     if windows?
       static_reqs_in_file = "#{windows_safe_path(project_dir)}\\datadog_checks_base\\datadog_checks\\base\\data\\#{agent_requirements_in}"
-      static_reqs_out_file = "#{windows_safe_path(project_dir)}\\#{filtered_agent_requirements_in}"
+      static_reqs_out_folder = "#{windows_safe_path(project_dir)}\\"
+      static_reqs_out_file = static_reqs_out_folder + filtered_agent_requirements_in
     else
       static_reqs_in_file = "#{project_dir}/datadog_checks_base/datadog_checks/base/data/#{agent_requirements_in}"
-      static_reqs_out_file = "#{project_dir}/#{filtered_agent_requirements_in}"
+      static_reqs_out_folder = "#{project_dir}/"
+      static_reqs_out_file = static_reqs_out_folder + filtered_agent_requirements_in
     end
 
     # Remove any blacklisted requirements from the static-environment req file
     requirements = Array.new
+
+    # Creating a hash containing the requirements and requirements file path associated to every lib
+    requirements_custom = Hash.new()
+    specific_build_env.each do |lib, env|
+      requirements_custom[lib] = {
+        "req_lines" => Array.new,
+        "req_file_path" => static_reqs_out_folder + lib + "-py2.in",
```

Comment:
Nit: the `piptools compile` output file (`#{install_dir}/agent_#{lib}_requirements-py2.txt` / `#{windows_safe_path(install_dir)}\\agent_#{lib}_requirements-py2.txt`) could also be defined here to avoid having to repeat it later, with something like:
`"piptools_file_path" => static_reqs_out_folder + "agent_" + lib + "_requirements-py2.txt"`)

Same for the py3 file.